### PR TITLE
PoC of atlasmap e2e testing 

### PR DESCRIPTION
**Pull request by mmelko**

### CODE REVIEW

File: datamapper-e2e/src/test/resources/atlas-itests.properties [hunk 1]
```diff
@@ -0,0 +1,8 @@
+http.host=localhost
+http.port=8586
+https.host=localhost
+https.port=8443
+https.keystore=ssl.keystore
+https.keystore.password=atlasmap
+spring.resources.static-locations=/Users/mmelko/RH_WORK/Work/Test/syndesis/atlasmap-qe/datamapper-e2e/src/main/resources/static/
```
Comment (dsimansk) [inline datamapper-e2e/src/test/resources/atlas-itests.properties]: This path is probably not very generic :)

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+*build/*
+*/target/*
+*/node
```
Comment (dsimansk) [inline .gitignore]: Add binary files like `datamapper-e2e/src/.DS_Store`

File: datamapper-e2e/src/test/resources/atlas-itests.properties (hunk 1 above)
Comment (mmelko) [reply datamapper-e2e/src/test/resources/atlas-itests.properties]: it's even not used :)

**Comment by igarashitm** [comment]
LGTM :+1: 
We may want to use `io.atlasmap.qe` for the package name? as it's under atlasmap organization.

**Comment by igarashitm** [comment]
@mmelko Do you want me to push the approve button? you can merge by yourself as an owner. This is a honorable first meaningful commit for this repo :-)

**Pull request merged**